docs/index.md

```diff
@@ -92,6 +92,25 @@ Goes well with the [Rust Enhanced package](https://github.com/rust-lang/rust-enh
 Alternatively, a newer [rust-analyzer](https://github.com/rust-analyzer/rust-analyzer) server is under development, also supported by LSP.
 
 
+### R<a name="r"></a>
+
+Requires installation of the `languageserver` package from CRAN:
+
+`install.packages("languageserver")`
+
+See the [CRAN mirrored package on GitHub](https://github.com/cran/languageserver) for more information and up-to-date installation instructions.
+
+```json
+"rlang":
+{
+  "command": ["R", "--slave", "-e", "languageserver::run()"],
+  "scopes": ["source.r"],
+  "syntaxes": ["Packages/R/R.sublime-syntax"],
+  "languageId": "r"
+}
+```
+
+
```

Comment:
Does this still need documentation if it's going to be part of the default clients? Should the documentation not  just say to `enable` the language server?

Reply:
Good point, was following other examples in the docs but realised they weren't all just repeating defaults in `clients`, will change to as you say